src/cloud/docker/docker-manage-cron-jobs.md

```diff
@@ -19,6 +19,14 @@ The Magento cron implementation has the following limitations:
 -  The `setup:cron:run` and `cron:update` commands are not available on Cloud and Docker for Cloud environments
 -  In the Docker environment, cron works only with the CLI container to run the `./bin/magento cron:run` command
 
+By default, cron container is not present in your Docker development environment to improve overall performance.
+
+## Add the cron container
```

Comment:
Add intro sentence: 

``` You can add the Cron container to the build configuration by  including the `--with-cron` option when you build the Docker environment.```

Reply:
It might be helpful to have a use case here --an example of when I would need to include cron, or a statement that explains why I might want to add the cron container.

Reply:
If the cron is not enabled by default, I don't think we need to include the "Disable the Cron container" section.

Reply:
Fixed